components/atoms/SearchBar.js

```diff
@@ -5,23 +5,23 @@ import PropTypes from "prop-types";
  */
 export function SearchBar(props) {
   return (
-    <div className="inline-flex" data-cy={props.dataCy}>
+    <div className="inline-flex searchBox" data-cy={props.dataCy}>
       <form
-        className="inline-flex border border-solid border-gray-light-100"
+        className="inline-flex w-full searchBox border border-solid border-gray-light-100 "
         onSubmit={props.onSubmit}
       >
```

Comment:
So the search input button should always be on the right side of the input (or container). Right now, it is slightly outside of it on big screens, and then on mobile it's almost centred.

<img width="308" alt="Screen Shot 2021-04-20 at 15 52 32" src="https://user-images.githubusercontent.com/2454380/115456375-18313380-a1f1-11eb-99b2-6f0fa9c01aa2.png">
☝️ here you can see a gap between the button and the input

<img width="496" alt="Screen Shot 2021-04-20 at 15 52 40" src="https://user-images.githubusercontent.com/2454380/115456376-18c9ca00-a1f1-11eb-8648-2e7391866045.png">
☝️ here you can see that it is centred

Reply:
is this on chrome?  firefox looked ok when i test this

Reply:
This was on Edge, yeah, which uses Chrome's engine these days.